config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/PolicyMetadata.java

```diff
@@ -0,0 +1,31 @@
+package com.ca.apim.gateway.cagatewayconfig.beans;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.Set;
+
+/**
+ * Metadata file for Policy entities
+ */
+
+@JsonInclude
+public class PolicyMetadata {
```

Comment:
I expect to populate few additional fields for the policy. 

Document: https://docs.google.com/document/d/17x1AnwYQc2mF6AENcCGYfNN0SjLtOyh4lZ8nj0VIgLA/edit
Section: Tracking dependencies for policies

If it is out of scope for this PR or story, please ignore it for now.